### DISCUSSION TITLE
Is there a example for lldap_config.toml that is always up to date?

### DISCUSSION THREAD

**Discussion by kastl-ars** [lldap/lldap#990] [Q&A] [answered]
Hi,

is there an example `lldap_config.toml` that is always "up to date", i.e. contains all the options that the current version supports?

I only found the [`lldap_config.docker_template.toml`](https://github.com/lldap/lldap/blob/main/lldap_config.docker_template.toml) that seems to have more docker-specific content?

Thanks in advance,
Johannes

**Comment by nitnelave** [comment, marked as answer]
That's the one! Some of the values are tuned to docker, that's true, but the keys and documentation are comprehensive.

**Reply by kastl-ars** [reply]
OK, I'll use that then.

**Reply by kastl-ars** [reply]
Thanks for the quick reply!
